tests/integration/test_dataflow_integration.py

```diff
@@ -1,25 +1,33 @@
 import json
 import subprocess
+import sys
 import tempfile
 import time
 from importlib.metadata import version
+from pathlib import Path
 
 import pytest
 import xarray as xr
 from packaging.version import parse as parse_version
 
+TEST_DATA_DIR = Path(__file__).parent.parent / "test-data"
+
 
 def test_dataflow_integration():
+    python_version = (
+        f"{sys.version_info.major}.{sys.version_info.minor}.{sys.version_info.micro}"
+    )
     pfr_version = parse_version(version("pangeo-forge-recipes"))
     if pfr_version >= parse_version("0.10"):
-        recipe_version_ref = str(pfr_version)
+        recipe_version_ref = "0.10.x"
```

Comment:
reviewer note: 
given that integrate/unit tests [only test beginning and end range](https://github.com/pangeo-forge/pangeo-forge-runner/blob/main/.github/workflows/unit-test.yml#L17-L23) of versions I made the unilateral decision to just have `0.[9||10].x` listed in test-data which I think works fine